src/backend/src/services/tasks.services.ts

```diff
@@ -37,27 +37,32 @@ export default class TasksService {
   ): Promise<Task> {
     const requestedWbsElement = await prisma.wBS_Element.findUnique({
       where: { wbsNumber: wbsNum },
-      include: { project: { include: { team: { ...teamQueryArgs }, wbsElement: true } } }
+      include: { project: { include: { teams: { ...teamQueryArgs }, wbsElement: true } } }
     });
     if (!requestedWbsElement) throw new NotFoundException('WBS Element', wbsPipe(wbsNum));
     if (requestedWbsElement.dateDeleted) throw new DeletedException('WBS Element', wbsPipe(wbsNum));
     const { project } = requestedWbsElement;
     if (!project) throw new HttpException(400, "This task's wbs element is not linked to a project!");
 
-    const { team } = project;
-    if (!team) throw new HttpException(400, 'This project needs to be assigned to a team to create a task!');
+    const { teams } = project;
+    if (!teams) throw new HttpException(400, 'This project needs to be assigned to a team to create a task!');
```

Comment:
I think u need to check whether the array is empty, not just whether it exists here